EIPS/eip-4950.md

```diff
@@ -0,0 +1,90 @@
+---

```

Comment:
Please use unix-style line endings (LF).

Reply:
How??
I am using the online editor and don't see any option to change the encoding

Reply:
Don't use the online editor. I recommend using VSC.

Reply:
I think it is fixed now